modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -78,13 +82,25 @@ static inline DST mul(SRC1 x, SRC2 y, float scale=1)
 }
 
 template<typename DST, typename SRC1, typename SRC2>
-static inline DST div(SRC1 x, SRC2 y, float scale=1)
+static inline
+typename std::enable_if<!std::is_same<DST, float>::value, DST>::type
+div(SRC1 x, SRC2 y, float scale=1)
 {
-    // like OpenCV: returns 0, if y=0
+    // like OpenCV: returns 0, if DST type=uchar/short/ushort and divider(y)=0
     auto result = y? scale * x / y: 0;
     return saturate<DST>(result, rintf);
 }
 
+template<typename DST, typename SRC1, typename SRC2>
+static inline
+typename std::enable_if<std::is_same<DST, float>::value, DST>::type
+div(SRC1 x, SRC2 y, float scale = 1)
+{
+    // like OpenCV: returns inf/nan, if DST type=float and divider(y)=0
+    auto result = scale * x / y;
+    return saturate<DST>(result, rintf);
+}
+
 template<typename DST, typename SRC1, typename SRC2>
 static inline DST divr(SRC1 x, SRC2 y, float scale=1)
```

Comment:
> divr

has we same flow, but it is still not updated.

Reply:
Within current task this wasn't and won't be updated. I added SIMD for Div kernel only.

Reply:
BTW, `divr()` function is declared but not used anywhere in `gfluidcore.cpp`